packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -123,6 +123,8 @@ describe( 'Writing Flow', () => {
 		expect( activeBlockName ).toBe( 'core/column' );
 		// Arrow down to reach second Column block
 		await page.keyboard.press( 'ArrowDown' );
+		activeBlockName = await getActiveBlockName();
+		expect( activeBlockName ).toBe( 'core/column' );
```

Comment:
The e2e passes consistently if we check intermediate block names and thus introduce a small delay. That explains why this particular check failed while the others pass (the all have checks after each key press). Ideally we do find what's causing presses in quick succession to fail.

Reply:
Ok, that doesn't seem to do it for Travis! It does seem like something is taking more time than it should, perhaps some excessive re-rendering.

Reply:
>I guess the best way to workaround it right now would be to throw in some page.waitForSelectors and page.waitForXPaths.

It seems that @ZebulanStanphill idea is correct, but doesn't apply here because we haven't inserted anything. Your change here is valid.

>Ideally we do find what's causing presses in quick succession to fail.

Yes, it might even be a race condition within jest and `press`. But probably would be from GB :)

Reply:
Doing a few preparatory PRs, so we can isolate the source of the problem better.

Reply:
There's another test with inner blocks that fails, so definitely something about the rendering has changed here.